packages/backend/src/core/reports/createTotalSupplyFormulaReports.ts

```diff
@@ -1,15 +1,14 @@
 import { assert, AssetId, ChainId, ProjectId, Token } from '@l2beat/shared-pure'
 
-import { BalanceRecord } from '../../peripherals/database/BalanceRepository'
+import { CirculatingSupplyRecord } from '../../peripherals/database/CirculatingSupplyRepository'
 import { PriceRecord } from '../../peripherals/database/PriceRepository'
 import { ReportRecord } from '../../peripherals/database/ReportRepository'
 import { TotalSupplyRecord } from '../../peripherals/database/TotalSupplyRepository'
 import { BalancePerProject, createReport } from './createReport'
 
-export function createEBVReports(
+export function createTotalSupplyFormulaReports(
```

Comment:
This one should be named differently, it is used in a formula agnostic way right now